css/style.css

```diff
@@ -0,0 +1,180 @@
+body {
+  margin: 10px 30px;
+  font-family: inherit;
+  background-color: #eceef1;
+}
```

Comment:
Please consider using "Sans-serif" as the font-family in the body, that way you don't have to specify a different font for each section.